### PR TITLE
BZ2111793: Improved OCM text in section 24.6 of Installation guide

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.9 +

Issue:
[BZ2111793](https://bugzilla.redhat.com/show_bug.cgi?id=2111793)

Link to docs preview:
[Section 24.6 of Validating an installation](https://55467--docspreview.netlify.app/openshift-enterprise/latest/installing/validating-an-installation.html#reviewing-cluster-status-from-the-openshift-cluster-manager_validating-an-installation)

QE review:
- [ ] QE has approved this change.

Additional information:
* For more details about the information requested in this template, see: [Section 24.6](https://access.redhat.com/documentation/en-us/openshift_container_platform/4.10/html/installing/validating-an-installation#reviewing-cluster-status-from-the-openshift-cluster-manager_validating-an-installation) of Installation guide

### CODE REVIEW

**Comment by dfitzmau** [comment]
Reviewers. See the following image to validate the change in procedural steps:

Screenshot from 2023-02-02 14-07-08

File: modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc [hunk 1]
```diff
@@ -6,22 +6,22 @@
...
+From the {product-title} web console, you can review detailed information about the status of your cluster on {cluster-manager}.
 
 .Prerequisites
 
 * You have access to the cluster as a user with the `cluster-admin` role.
 
 .Procedure
 
-. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *{cluster-manager}* to open the *Overview* page for the cluster in {cluster-manager-url}.
+. In the *Administrator* perspective, navigate to *Home* -> *Overview* -> *Details* -> *Cluster ID* -> *{cluster-manager}* to open the *Clusters* page for the cluster in {cluster-manager-url}.
```
Comment (dayleparker) [inline modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc:17]: A few small issues -

1. Does step 1 take you directly into the details for a cluster (ie. this view https://console.redhat.com/openshift/details/s/1U9JqcRiAXsrylGHJAGHJGHGHvS6#overview) , or the general list of all the clusters (this page https://console.redhat.com/openshift) ? If the navigation path mentioned takes you directly into a single cluster's Overview page, I'd call it 'the cluster details page' or 'the cluster Overview page' - we use wording like that in the OCM guide - https://access.redhat.com/documentation/en-us/openshift_cluster_manager/2022/html-single/managing_clusters/index#assembly-managing-clusters

2. I know this isn't part of this PR, but the attribute {cluster-manager-url} needs to be fixed as this is rendering as an incorrect product/service name. 

I'd suggest writing it as "in the Hybrid Cloud Console" with same URL for the hyperlink linked to the words "Hybrid Cloud Console". I think it's more important to emphasize the Hybrid Cloud Console (and the name OpenShift Cluster Manager might change or disappear in the upcoming changes to the console.)

@aldiazRH - please provide a sanity check for me :)

**Comment by dayleparker** [comment]
Hi Darragh,
Thanks for making this update! In checking the preview of the doc, there were two things in need of an edit. Let me know if you need additional details or explanation!

Also, Alberto Diaz is now the writer who owns the OpenShift Cluster Manager docs, so I would recommend he provides the final review, in case any of my knowledge is outdated. Thanks so much :)

Cheers,
Dayle

File: modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc (hunk 1 above)
Comment (dfitzmau) [reply modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc:17]: Thanks, @dayleparker .

1. I see a Clusters page that lists all the clusters, so I won't change the name in this instance. Unless @aldiazRH , the production environment is different?

2. Sadly, 3266 references to the cluster-manager-url manager exist. If I change this attribute, I would need to spend a lot of time checking all instances read correctly. If the OCM Hybrid Cloud Console name were to change, the justification for doing a wider sweep would be easier. 

@sjstout , what do you think? Keep `:cluster-manager-url: link:https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console]` the same in the `common-attributes.adoc` file? I do agree that the link name does not directly align with what displays on the Hybrid Web Console.

Comment (aldiazRH) [reply modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc]: @dfitzmau 

1. I see the same, so I'm happy to leave it as is. 

2. I agree with the problematic naming, but I also agree that the change for that is out of scope for this specific PR. That would have to be a separate issue, which I could take on, as I'm the one working on OCM docs.

Comment (dfitzmau) [reply modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc]: Thanks, @aldiazRH . Just wondering if the following note should now be removed (based on my proposed update)?

Screenshot from 2023-02-02 17-00-48

Thanks for taking on part 2!

Would you happen to know an engineer that could review my proposed update?

Comment (aldiazRH) [reply modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc]: Yes, I agree, the note is no longer needed in this situation. :+1: 

As for eng, I would reach out to Dave Taylor or Kim Doberstein to see if they, or someone on their team can take a look. That'd be my first guess, anyway.

**Comment by dtaylor113** [comment]
Hi @dfitzmau, I'm a little confused, are these instructions how to get to the OCM UI (https://console.redhat.com/openshift) from within OCP console?

Is '2. In the Clusters page...' supposed to be the OCM's cluster details page?

If so, then it could be phrased like '2. On the Clusters page, the Overview tab will show...' as the 'Overview' tab will be the default selection on the Cluster (details) page.   Cluster history (one of the doc's bullet points) is actually under the 'Cluster history' tab:

image

**Comment by dfitzmau** [comment]
> Hi @dfitzmau, I'm a little confused, are these instructions how to get to the OCM UI (https://console.redhat.com/openshift) from within OCP console?
> 
> Is '2. In the Clusters page...' supposed to be the OCM's cluster details page?
> 
> If so, then it could be phrased like '2. On the Clusters page, the Overview tab will show...' as the 'Overview' tab will be the default selection on the Cluster (details) page. Cluster history (one of the doc's bullet points) is actually under the 'Cluster history' tab:
> 
> image

Thanks for the review, @dtaylor113 . Yes that is the intended route for the user to navigate to the cluster's page. I updated the content based on your feedback. Please let me know your thoughts.

**Comment by dtaylor113** [comment]
Hi @dfitzmau, I'm still unclear on where user lands when linking-and-launching from OCP to OCM.

You wrote:

1. In the Administrator perspective, navigate to Home → Overview → Details → Cluster ID → {cluster-manager} to open the Clusters page in {cluster-manager-url}.
2. On the Clusters page, click your cluster’s name. The Overview tab for your cluster opens...

User starts off in OCP cluster details page and clicks on link to OCM.  I would think they would land directly on OCM's cluster details page, so "...Clusters page, click your cluster’s name..." would not be needed.  -Clicking on a cluster's name would only be applicable if they land in OCM's cluster listing page and not the cluster's details page.

**Comment by dtaylor113** [comment]
I also do not understand the later steps?  Are these in OCM or OCP UI?:

"4. Navigate to the Observe page...to find:
-  A list of any issues that have been detected
- A list of alerts that are firing
- The cluster Operator status and version
- Cluster resource usage"

OCM does not have a 'Observe' page AFAIK, I think this is now 'Monitoring':
image

"5. Navigate to the Insights page...to find:
- Potential issues that your cluster might be exposed to, categorized by risk level
- Health-check status by category"

I think this is OCM's main 'Overview' tab?
image

**Comment by dtaylor113** [comment]
Hi @dfitzmau, latest looks good, however the last two bullet points of Step 2 needs to be removed:

- Potential issues that your cluster might be exposed to, categorized by risk level
- Health-check status by category

These would be found by clicking on OCM's left hand navigation plane and selecting the 'Overview' tab.

**Comment by dfitzmau** [comment]
> Hi @dfitzmau, latest looks good, however the last two bullet points of Step 2 needs to be removed:
> 
> * Potential issues that your cluster might be exposed to, categorized by risk level
> * Health-check status by category
> 
> These would be found by clicking on OCM's left hand navigation plane and selecting the 'Overview' tab; they are not located on the cluster details page.

Thanks, @dtaylor113 . I updated the content based on your feedback. Please let me know your thoughts.

**Comment by dtaylor113** [comment]
Thanks @dfitzmau, perfect :-) LGTM

File: modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc [hunk 2]
```diff
@@ -35,18 +30,21 @@ Alternatively, you can navigate to {cluster-manager-url} directly and select you
...
 +
 * A list of alerts that are firing
 +
 * The cluster Operator status and version
 +
-* Cluster resource usage
+* The cluster's resource usage
 
-. Navigate to the *Insights* page to review the following information provided by Red Hat Insights:
+. _Optional:_ You can view information about your cluster that Red Hat Insights collects by navigating to the *Overview* menu. From this menu you can view the following information:
```
Comment (kcarmichael08) [inline modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc:47]: Suggest adding comma after introductory phrase per IBM Style.
```suggestion
. _Optional:_ You can view information about your cluster that Red Hat Insights collects by navigating to the *Overview* menu. From this menu, you can view the following information:
```

Comment (kcarmichael08) [reply modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc:47]: I don't think "Optional" is italicized normally, but I can't find the specific guidance in the style guides.

**Review by kcarmichael08 — COMMENTED** [review]
Hi, I had a couple of suggestions - LGTM!

**Comment by kelbrown20** [comment]
@dfitzmau Hello Darragh! I looks like this PR still needs a QE review before this can be merged. I see that there is a QE contact [mentioned in the BZ](https://bugzilla.redhat.com/show_bug.cgi?id=2111793#c3) that could take a look! 

Thank you!

**Comment by dfitzmau** [comment]
> @dfitzmau Hello Darragh! It looks like this PR still needs a QE review before this can be merged. I see that there is a QE contact [mentioned in the BZ](https://bugzilla.redhat.com/show_bug.cgi?id=2111793#c3) that could take a look!
> 
> Thank you!

Thanks, @kelbrown20 . I'll request a QA review.

**Comment by dfitzmau** [comment]
Hello @jiajliu .

Would you be OK to QE review this PR that stems from [Red Hat Bugzilla – Bug 2111793](https://bugzilla.redhat.com/show_bug.cgi?id=2111793)

**Comment by dfitzmau** [comment]
Hello @xueli181114 . I see that you were assigned as the new QE contact for https://bugzilla.redhat.com/show_bug.cgi?id=2111793 . Would you be OK to review this PR?

**Comment by jmekkatt** [comment]
Hi @dfitzmau , During QE review, it has found that 
> "The cluster ID, status, type, location, and the provider name"

But we don't see any location on overview tab instead it is Region.
image
Probably we want to correct this as well. What do you think? Thank you.

**Comment by jmekkatt** [comment]
Thanks for the change @dfitzmau and LGTM

**Pull request merged**